CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+  * Shows different icon for category and scorecard in check result
+  * Fixes bug where scorecard section showed when an empty array returned
```

Comment:
does it? 🤔

Reply:
...I think this line of the changelog is for [this other PR](https://github.com/OpsLevel/backstage-plugin/pull/58/files)

Reply:
Yup. I'll merge both and the world will *never know*